charts/pega/values-large.yaml

```diff
@@ -407,3 +409,38 @@ hazelcast:
   # username,password parameters in External Secrets Manager, and enter the external secret for the credentials
   # make sure the keys in the secret should be HZ_CS_AUTH_USERNAME , HZ_CS_AUTH_PASSWORD
   external_secret_name: ""
+
+# Stream (External Kafka) settings.
+stream:
+  # Disabled by default, you must not use stream tier when this setting is enabled.
+  enabled: false
+  # Provide external kafka broker urls.
+  bootstrapServer: ""
+  # Provide Security Protocol used to communicate with brokers. Supported values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL.
+  securityProtocol: PLAINTEXT
+  # If required, provide a Kubernetes secret name that holds the trustStore certificate
+  # Please add the Kubernetes secret name in 'global.certificatesSecrets' too
+  # Pega supports only Java Key Store (.jks) format for truststore.
+  trustStore: ""
+  # If required provide trustStorePassword value in plain text.
+  trustStorePassword: ""
+  # If required, provide a Kubernetes secret name that holds the keyStore certificate.
+  # Please add the Kubernetes secret name in 'global.certificatesSecrets' too.
+  # Pega supports only Java Key Store (.jks) format for keystore.
+  keyStore: ""
+  # If required, provide keyStore value in plain text.
+  keyStorePassword: ""
+  # If required, provide jaasConfig value in plain text.
+  jaasConfig: ""
+  # If required, provide SASL Mechanism, Supported values are: PLAIN, SCRAM-SHA-256, SCRAM-SHA-512.
+  saslMechanism: PLAIN
+  # By default, topics originating from Pega Platform have the pega- prefix,
+  # so that it is easy to distinguish them from topics created by other applications.
+  # You can configure the name pattern to customize topic names per environment.
+  streamNamePattern: "pega-{stream.name}"
```

Comment:
Is this supposed to be changed by the User? Or default can be this way having substitutes?

Reply:
Users may change it otherwise the current value is the default value